contracts/account/Administrator.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+contract Administrators {
+	// administrators that can handle grid token allocations, treasury, etc
+	// future: implement voting triad (or other governance option)
+	address[3] public admin;
+
+	/// @notice define token contract administrators
+	/// @dev setup administrator in a way that only one of the 3 admins can perform token actions
+	/// @param _admin array of administators
+	function setAdminstrators(
+		address[3] memory _admin
+	) public {
+		require(_admin[0] != _admin[1] && _admin[0] != _admin[2] && _admin[1] != _admin[2], "Admins must be different addresses");
```

Comment:
This could be changed to a modifier like onlyAdmin

Reply:
I did some adjustment preventing the 2nd call